tests/test__ped_partition.py

```diff
@@ -177,18 +182,39 @@ def runTest(self):
         self._part.set_name("blah")
         self.assertEqual(self._part.get_name(), "blah")
 
-@unittest.skip("Unimplemented test case.")
-class PartitionIsBusyTestCase(unittest.TestCase):
+class PartitionIsBusyTestCase(RequiresPartition):
+    def setUp(self):
+        RequiresPartition.setUp(self)
+        self.mountpoint = None
+
+    def doMount(self):
+        self.mountpoint = tempfile.mkdtemp()
+        os.system("mount -o loop %s %s" % (self.path, self.mountpoint))
+
+    def mkfs(self):
+        os.system("/sbin/mkfs.ext2 -F -q %s" % self.path)
+
+    def tearDown(self):
+        if self.mountpoint:
+            os.system("umount %s" % self.mountpoint)
+            os.rmdir(self.mountpoint)
+        RequiresPartition.tearDown(self)
+
     def runTest(self):
-        # TODO
-        self.fail("Unimplemented test case.")
+        # partitions aren't busy until they're mounted.
+        self.assertFalse(self._part.is_busy())
+
+# TODO:  need to figure out how to make a loopback device look mounted to
+# libparted
+#        self.mkfs()
+#        self.doMount()
+#        self.assertTrue(self._part.is_busy())
+
 
 class PartitionGetPathTestCase(RequiresPartition):
     def runTest(self):
         self.assertNotEqual(self._part.get_path(), "")
 
-@unittest.skip("Unimplemented test case.")
-class PartitionStrTestCase(unittest.TestCase):
+class PartitionStrTestCase(RequiresPartition):
     def runTest(self):
-        # TODO
-        self.fail("Unimplemented test case.")
+        self.assertTrue(str(self._part).startswith("_ped.Partition instance"))
```

Comment:
Can this not just be self.assertTrue(self._part is _ped.Partition)?  I don't see the need for the str() conversion and startswith() test here.